src/main/java/org/primefaces/renderkit/InputRenderer.java

```diff
@@ -112,4 +112,19 @@ protected void renderARIARequired(FacesContext context, UIInput component) throw
         }
     }
 
+    /**
```

Comment:
If it's really meant for combobox, what about putting it in `SelectRenderer`? Or you think it might be needed by renderer inheriting from InputRenderer?